pkgs/by-name/be/bevy_skein/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  stdenv,
+  alsa-lib,
+  udev,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "bevy_skein";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "rust-adventure";
+    repo = "skein";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-pyO3/jMzlbjxv7XP9xtwR0KhXMmcQITzmBpDebP2d5Q=";
+  };
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-Q/6s4nLKrIfbJ+rJCsEiT5XnEp3mAS9K74L0iR4LYdU=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs =
+    [ openssl ]
+    ++ lib.optionals stdenv.isLinux [
```

```suggestion
    ++ lib.optionals stdenv.hostPlatform.isLinux [
```

Missed this in the previous review, sorry! After this it should be good to go.

---

For explanation: this is relevant for cross compiling, where `stdenv.hostPlatform` is the platform we are building **for**, and `stdenv.buildPlatform` is the platform we are building **on**